### PR TITLE
(WIP) Upgrade to queens

### PR DESCRIPTION
**Pull request by jovial**
Not all of the version tags have been updated

### CODE REVIEW

**Comment by jovial** [comment]
@dougszumski Not sure what need to be done in regards to monitoring

**Comment by dougszumski** [comment]
Still working on queens support for Monasca.

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -110,7 +110,7 @@ kolla_build_blocks:
     # Install a locally modified version of networking-mlnx with support for
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml:110]: Looks like I added a stackhpc/1.0.0.1 tag, let's use that.

File: etc/kayobe/kolla/globals.yml [hunk 2]
```diff
@@ -2,6 +2,9 @@
 # Add extra Kolla global configuration here.
 rabbitmq_monitoring_user: "monitoring"
 
+# Although Kolla-Ansible provides full support for Fernet tokens, it still
+# defaults to UUID.  This setting overrides K-A and brings it in-line with
+# Keystone's default.
 keystone_token_provider: "fernet"
 keystone_tag: 6.0.0.0
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: Shall we remove this and below to allow keystone and magnum to use the new images?

**Review by markgoddard — APPROVED** [review]

**Review by markgoddard — APPROVED** [review]

**Pull request merged**